crates/nvim-api/src/ffi/buffer.rs

```diff
@@ -118,8 +109,7 @@ extern "C" {
     ) -> Array;
 
     // https://github.com/neovim/neovim/blob/master/src/nvim/api/buffer.c#L1086
-    pub(crate) fn nvim_buf_get_name(buf: BufHandle, err: *mut Error)
-        -> String;
+    pub(crate) fn nvim_buf_get_name(buf: BufHandle, err: *mut Error) -> Str;
```

Comment:
This might be a breaking change. `Str` might need to implement some traits. Maybe by duplicating some code from [string.rs](https://github.com/noib3/nvim-oxi/blob/29bdd3383eedc24bf86d733fa70dbfa21ac91d82/crates/nvim-types/src/string.rs)
The main different between `Str` and `String` is the former  doesn't implement custom `Drop` to drop ffi memory.